plugins/inputs/mcrouter/README.md

```diff
@@ -0,0 +1,102 @@
+# Mcrouter Input Plugin
+
+This plugin gathers statistics data from a Mcrouter server.
+
+### Configuration:
+
+```toml
+# Read metrics from one or many mcrouter servers.
+[[inputs.mcrouter]]
+  # An array of address to gather stats about. Specify an ip on hostname
+  # with optional port. ie localhost, 10.0.0.1:11211, etc.
+  servers = ["localhost:11211"]
+  # An array of unix mcrouter sockets to gather stats about.
+  # unix_sockets = ["/var/run/mcrouter.sock"]
```

Comment:
Let's use a single list of this style:
```
servers = ["tcp://localhost:11211", "unix:///var/run/mcrouter.sock", "udp://localhost:11211"]
```

We don't need to add support for UDP at first, assuming it is even available with mcrouter, but this will give us a nice way to add it later.

Reply:
I've updated to use a single array of servers and pull the protocol from the connection string.

(mcrouter does not support UDP connections)